README.md

```diff
@@ -140,27 +178,66 @@ _After_ you have ***contributed*** to the project,
 _e.g: by commenting on or creating an issue_, <br />
 -->
 If you've found the project interesting, helpful or useful in _any way_,
-_please_ share the love by starring it! <br />
+_please_ ***star*** the repository on GitHub! <br />
 Stars show _other_ members in the developer community
 that it's a _**worthwhile project** or **learning resource**_
 and one that can offer value to other developers like you! 🌟
 
 
 <br />
 
-## Questions?
+# Questions?
+
+### _Why_ Does the Issue _Need_ to be _Validated_? >> One Word: _Communication_
 
+This _may initially feel_ a bit "_bureaucratic_" but
+we _urge_ you _not_ to think of it as <br />
+["_jumping through hoops_"](https://en.wiktionary.org/wiki/jump_through_hoops)
+but _rather_, _we_ think of ***issue validation*** as:
+
++ ***ensures*** that **everyone** on the project
+is ***aware*** of the issue/idea/story.
+
++ ***saves you time*** by ***avoiding*** ***duplicating effort***!
+> The **Product Owner** (**PO**) or **Project Lead** (**PL**)
+has _detailed knowledge_ of the _current_ issue backlog
+and all _previous_ ("closed") issues. <br />
+They know _how_ features/functionality _should_ work and can
+validate from the _user's_ perspective. <br />
+***Most importantly*** the PO/PL ***knows*** if someone ***else***
+is ***already*** working on the issue ... <br />
+And, _sometimes_ the _exact_ same issue has ben "***solved before***"
+in a _different_ project, so ***hours/days*** of effort can be saved!!
+
+
++ ***someone else*** might have **insight** they can share on the issue! <br />
+> e.g: if you have a **question** or are stuck on a challenge/problem,
+[_**share** your **thoughts** and/or **progress**
+towards "**solving**" it in **issue comments**_](https://github.com/dwyl/learn-nightwatch/issues/8#issuecomment-264734792)
+you _never_ know when you will get _insight_ from a ***complete stranger***!
+
++ ***capturing information in issues*** _maintains_ ***relevance and focus***.
+> the discussion is not "***lost***"
+the way it would be in "_chat_"
+and discussion are grouped by relevant topic.
+
++ ***clearly document*** the project so anyone _joining_ the team
+_later_ on can see the "_history_".
+> We have found this to be _invaluable_ when people are _learning_
+why and how something works the way it does!
 
 
 ### How _Long_ Does it take for an Issue to be _Validated_?
 
 We ***aim*** to respond to new issues within _**minutes**_
 but during busy work days this can take up to ***a few hours***... <br />
-(_**thank you** for your **patience**!_)
+(_**thank you** for your **patience**!
+  **please** help us by giving as much useful information
+  as possible in the issue description._)
 
 ![thinking-please-wait](https://cloud.githubusercontent.com/assets/194400/22228969/9add48f8-e1cc-11e6-8cdc-ef9b30eac846.jpg)
```

Comment:
please substitute it for something better or simply remove it completely.
I just got "_frustrated_" by the text-heavy previous version of this doc and went image-crazy.
I'm sure someone `else` can tidy this up in the next couple of iterations. 👍